internal/databases/sqlserver/blob/server.go

```diff
@@ -94,6 +102,18 @@ func NewServer(folder storage.Folder) (*Server, error) {
 	if bs.crypter != nil {
 		bs.encryption = bs.crypter.Name()
 	}
+	c, err := lru.NewWithEvict(BlockReadCacheSize, func(k, _ interface{}) {
+		tracelog.DebugLogger.Printf("EVICT_CACHE: %s", k)
+	})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create lru cache: %v", err)
+	}
+	bs.readCache = c
+	go func() {
+		// to release memory after restore
+		time.Sleep(time.Minute)
+		bs.readCache.RemoveOldest()
```

Comment:
It is a little bit weird that you are going to remove _one_ record from the cache after one minute...

Reply:
This is poor man's  expiration for lru cache :)
When database is restored, cache is no longer needed, and we can free some memory.
But it's hard to say when exactly restore is done, so we just purge cache slowly one by one item.

Reply:
I think that in LRU cache old pages will be evicted without your explicit actions.
When we are talking about memory - I think that golang is greedy and won't return a byte to the operation system when you free some bytes in your app.

Reply:
They won't. LRU cache doesn't know about time and only evict items when new ones added.
Golang can return memory back to OS (unless it was fragmented, of course).

Reply:
Hm... It seems that you are right: golang authors say that since 1.12
>> The Go runtime now releases memory back to the operating system more aggressively, particularly in response to large allocations that can't reuse existing heap space.